.github/workflows/build-test.yml

```diff
@@ -158,6 +158,13 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 50
+    - name: Cache Composer dependencies
+      uses: actions/cache@v4
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
```

Comment:
What will be the output of `steps.composer-cache.outputs.dir` ???
I dont see an id `composer-cache` , there should be an step to set `dir` right?
Follow the link and see how actually the action suggests setting up cache. [Cache Composer Dependencies](https://github.com/shivammathur/setup-php?tab=readme-ov-file#cache-composer-dependencies)